Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs

```diff
@@ -62,6 +62,21 @@ public FourCC GetFormat()
         }
     }
 
+    public class TouchPositionTransformProcessor : IInputControlProcessor<Vector2>
+    {
+        public Vector2 Process(Vector2 value, InputControl control)
+        {
+#if UNITY_EDITOR
+            return value;
+#elif PLATFORM_ANDROID
+            return new Vector2(value.x, InputRuntime.s_Instance.screenSize.y - value.y);
+#else
+            return value;
+#endif
+        }
+    }
```

Comment:
This one I don't like so much. If there's no need for a processor, there simply shouldn't be a processor on the control. For now, let's take it and then refine it.